packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -1,52 +1,150 @@
 import classNames        from 'classnames';
-import { isArrayLike }   from 'mobx';
 import PropTypes         from 'prop-types';
 import React             from 'react';
 import { CSSTransition } from 'react-transition-group';
 import { Scrollbars }    from 'tt-react-custom-scrollbars';
-import Icon              from 'Assets/icon.jsx';
-import Items             from './items.jsx';
-import NativeSelect      from './native-select.jsx';
+import Icon              from 'Components/icon-arrow.jsx';
 import {
-    getDisplayText,
     getItemFromValue,
     getValueFromIndex,
     getPrevIndex,
-    getNextIndex }       from './helpers';
+    getNextIndex,
+    listPropType,
+}                        from './helpers';
+import Items             from './items.jsx';
+import NativeSelect      from './native-select.jsx';
+import SymbolSpan        from './symbol-span.jsx';
+import './dropdown.scss';
```

Comment:
@reza-fs for the components the one that you export (not subcomponents) please keep the css import into the index.js. in this PR will be only for `dropdown` and `popover`.